tools/wptrunner/wptrunner/wptmanifest/backends/base.py

```diff
@@ -1,4 +1,5 @@
 import abc
+import six
```

Comment:
Could we import the names we want directly i.e. `from six import iterkeys, iteritems, itervalues`?

Reply:
Fixed!